src/components/AccessibilityRequestsTable/index.tsx

```diff
@@ -67,12 +70,8 @@ const AccessibilityRequestsTable: FunctionComponent<AccessibilityRequestsTablePr
       },
       {
         Header: t('requestTable.header.testDate'),
-        accessor: 'testedAt',
-        Cell: ({ value }: any) => {
-          if (value) {
-            return formatDate(value);
-          }
-          return '';
+        accessor: (row: AccessibilityRequestsTableRow) => {
+          return `${row.relevantTestDate?.date || 'Not Added'}`;
```

Comment:
The string here would be best in the internationalization file for accessibility. I'm happy to walk you through that if ya want. Aside from that, yay for JS code!

Reply:
I'll take a whack at that!

Reply:
Done!